src/powershell/Public/Remove-FinOpsHubScope.ps1

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    Delete a Cost Management export and optionally data associated with the export.
+
+    .PARAMETER Id
+    Required Resource ID of the scope to remove.
+
+    .PARAMETER HubName
+    Optional. Name of the hub instance to update.
+
+    .PARAMETER RemoveData
+    Optional. Indicates whether to remove data for this scope from storage. Default = false
+
+    .EXAMPLE
+    Remove-FinOpsHubScope -Id "ResourceID of Scope" -HubName "Hub Name" 
+    Remove-FinOpsHubScope -Id "ResourceID of Scope" -HubName "Hub Name" -RemoveData
+    
+    .LINK
+    https://aka.ms/ftk/Initialize-FinOpsHubDeployment
+#>
+
+
+function Remove-FinOpsHubScope {
+    [CmdletBinding(SupportsShouldProcess)]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$Id,
+        [string]$HubName,
+        [switch]$RemoveData
+    )
+
+    try {
+        # Get all exports for the scope that are pointed to the included storage account and the msexports container
+        $exports = Get-FinOpsCostExport -Scope $Id  
+
+        # Delete the exports
+        foreach ($export in $exports) 
+        {
+            Remove-FinOpsCostExport -Scope $Id -Name $export.Name 
```

Comment:
We should probably tell it to delete its data, just in case we have anything left behind. This'll just simplify our cleanup.
```suggestion
            Remove-FinOpsCostExport -Scope $Id -Name $export.Name -RemoveData:$RemoveData
```

Reply:
Sure so we'd like to remove data in msexports and ingestion both. I'd change it.